js/customize-migrate.js

```diff
@@ -0,0 +1,53 @@
+/* global jQuery, wp */
+(function( $ ) {
+	'use strict';
+	var component = {
+		doingAjax: false,
+		postMigrationCount: 20
+	};
+	component.init = function() {
```

Comment:
All these JS methods should have docblocks and there should be spacing between them.